i18n/locales/en/common.json

```diff
@@ -337,6 +340,7 @@
   "Remove from bookmarks": "Remove from bookmarks",
   "Removed Votes": "Removed Votes",
   "Report the error via E-Mail": "Report the error via E-Mail",
+  "Report the error via email": "Report the error via email",
```

Comment:
Shouldn't be the same as the above line?

Reply:
right, I fixed it